symbulate/distributions.py

```diff
@@ -78,6 +77,9 @@ def __init__(self, p):
         self.xlim = (0, 1) # Bernoulli distributions are not defined for x < 0 and x > 1
  
     def draw(self):
+        """A function that takes no arguments and 
+            takes a single draw from the Bernoulli distribution."""
```

Comment:
I would word this as "...and returns a single draw..."

Ditto for all of the docstrings below.